### PR TITLE
LPS-188558

### PR DESCRIPTION
**Pull request by CaiocSiqueira**
https://liferay.atlassian.net/browse/LPS-188558

### CODE REVIEW

**Comment by CaiocSiqueira** [comment]
ci:test:sf

**Comment by CaiocSiqueira** [comment]
ci:test:search-solr:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/SolrSearch.testcase [hunk 1]
```diff
@@ -164,6 +164,32 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test ConcurrentReindexExecutionModeNotAvailableForSolr {
```
Comment (joshchong) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/SolrSearch.testcase:168]: Rename to `ConcurrentReindexNotAvailableOnSolr`

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/SolrSearch.testcase [hunk 2]
```diff
@@ -164,6 +164,32 @@ definition {
...
+		property custom.properties = "feature.flag.LPS-177664=true";
+		property portal.acceptance = "temp";
+
+		task ("Go to search admin and assert Solr is running") {
+			SearchAdministration.openSearchAdmin();
+
+			AssertTextEquals.assertPartialText(
+				locator1 = "SearchAdmin#SEARCH_ENGINE_VENDOR",
+				value1 = "Solr");
+		}
```
Comment (joshchong) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/SolrSearch.testcase:178]: Can remove this assertion.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/SolrSearch.testcase (hunk 1 above)
Comment (joshchong) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/SolrSearch.testcase:168]: Move entire test to Solr.testcase as that file is more appropriate for Solr specific use cases. SolrSearch.testcase is for use cases that are also relevant with Elasticsearch, but is just specifically tested on Solr

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/SolrSearch.testcase [hunk 3]
```diff
@@ -164,6 +164,32 @@ definition {
...
+
+		task ("Open index actions and assert Concurrent reindex is not available") {
+			SearchAdministration.openIndexActions();
+
+			AssertElementPresent(
+				locator1 = "Radio#ANY",
+				radioLabel = "All Instances");
```
Comment (joshchong) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/SolrSearch.testcase:185]: Can remove this

**Review by joshchong — CHANGES_REQUESTED** [review]

**Comment by CaiocSiqueira** [comment]
ci:test:sf

**Comment by CaiocSiqueira** [comment]
ci:test:search-solr:nocompile

**Comment by CaiocSiqueira** [comment]
ci:test:sf

**Comment by CaiocSiqueira** [comment]
ci:test:search-solr:nocompile

**Comment by CaiocSiqueira** [comment]
ci:test:sf

**Comment by CaiocSiqueira** [comment]
ci:test:search-solr:nocompile

**Pull request closed without merging**